style/pileup.css

```diff
@@ -4,11 +4,47 @@
  * You are encouraged to override anything you like.
  */
 
+.pileup-root {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+}
+
+.pileup-root > .track {
+  display: flex;
+  flex-direction: row;
+}
+.track-label {
+  flex: 0 0 100px;  /* fixed-width track labels */
+  text-align: right;
+  font-family: Helvetica Neue, Helvetica, Arial, sans;
```

Comment:
Let's use the 'freight-sans' typekit font we use in Cycledash (unless you want to keep it universal)
`font-family: "freight-sans-pro", verdana, helvetica, sans-serif;`

If you want to keep it Helvetica:
Multi-word fonts are usually wrapped in quotation marks, and 'sans-serif' is used as the catch-all:
`font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;`